src/kibana/components/doc_table/components/table_row.js

```diff
@@ -104,10 +105,10 @@ define(function (require) {
           }
 
           $scope.columns.forEach(function (column) {
-            newHtmls.push(cellTemplate({
+            newHtmls.push(noWhiteSpace(cellTemplate({
```

Comment:
I think I found a better fix. It still involves using `noWhiteSpace` in two places but I think its the right way to use it. Please hold while I commit.